### ISSUE TITLE
godoc improvement: some comments are shown as comments

### ISSUE THREAD

**Issue by mdomlad85** [golang/gddo#591]
When I go to type with a link (i.e. [https://golang.org/src/builtin/builtin.go#L255](example)) I got field list with comments instead of nicely formatted as it is on functions and structs. This should be done on any fields (constants too). Generally, the comment should never be presented as a comment but nicely formatted and connected to the part where it is commented.

**Comment by arnottcr** [comment]
I am a little unclear what issue you are describing. Can you rephrase, and if possible provide screenshots of behaviour?

That being said, it sounds like you could be complaining about the way struct fields are displayed:
> <a href="https://godoc.org/github.com/aws/aws-sdk-go/aws#WriteAtBuffer"><img src="https://user-images.githubusercontent.com/2526260/55689869-ae32f200-5979-11e9-9785-f676523093af.png"></a>

And you would like the fields broken out as hierarchical sub-elements, similar to funcs (even top level ones):
> <a href="https://godoc.org/github.com/aws/aws-sdk-go/aws#ReadSeekCloser"><img src="https://user-images.githubusercontent.com/2526260/55689929-70829900-597a-11e9-86f6-f5dbd8602910.png"></a>

Assuming that is the case, this feels like a proposal and is above my pay grade, however I would suggest you build a mock-up to better sell your position.
